docs/_posts/2014-08-07-quickstart.md

```diff
@@ -30,25 +30,25 @@ Quick Start
         pong
 
         $ curl -s "http://localhost:8081/healthcheck"
-        * deadlocks: OK
-        * emo-cassandra: OK
-          127.0.0.1(127.0.0.1):9160 879us
+        {"deadlocks":{"healthy":true},"ugc_global-cassandra":{"healthy":true,"message":"127.0.0.1(127.0.0.1):9160 124us"},...}
 
 4.  To erase the EmoDB data, simply delete the data folder:
 
         $ rm -rf bin/data/
         $ bin/start-local.sh
+{:.workflow}
 
 
 ### Quick Tutorial
 
-The following examples assume you have [jq](https://stedolan.github.io/jq/) or an equivalent (see Recommended
-Software below).  It is optional-- `jq .` just formats the JSON responses to make them easier to read.
+Normally you would start by creating roles and API keys to provide controlled access to EmoDB.  However, for the sake
+of keeping the tutorial simple we'll make all requests as the administrator, which in the default configuration provided
+by `start-local.sh` uses API key "local_admin".
 
 1.  Create a table in the System of Record.  Specify a "table template" with properties that will be returned with
     every object in the table:
 
-        $ curl -s -XPUT -H "Content-Type: application/json" \
+        $ curl -s -XPUT -H "Content-Type: application/json" -H "X-BV-API-Key: local_admin" \
```

Comment:
Why even add the complexity of adding the API-Key header. For the quick tutorial, isn't it better to just keep things as "anonymous"? And make a note that we allow anonymous access in the sample config for quick tutorial? Otherwise for consistency sake, we would have to add that in every one of our sample codes.